exercises/pythagorean-triplet/example.py

```diff
@@ -1,37 +1,61 @@
-def triplets_in_range(min, max):
-    result = set([x for b in
-                  range(4, max + 1, 4) for x in
-                  primitive_triplets(b)])
-    for x, y, z in set(result):
-        a, b, c = (2 * x, 2 * y, 2 * z)
-        while c <= max:
-            if a >= min:
-                result.add((a, b, c))
-            a, b, c = (a + x, b + y, c + z)
-    return set([(d, e, f) for d, e, f in result if d >= min and f <= max])
+from math import sqrt, ceil
 
 
-def maxMin(a, b):
-    return (a, b) if a > b else (b, a)
+def triplets_in_range(min, max):
```

Comment:
Ideally you shouldn't shadow the `min` and `max` builtins with these variable names. The empty slug.py file uses **range_start** and **range_end** to avoid that.

Reply:
Reasonable.